docs/reference/plugins/py-splashscreen.md

```diff
@@ -0,0 +1,65 @@
+
+# &lt;py-splashscreen&gt;
+
+This is one of the core plugins in PyScript, which is active by default. The splashscreen is the first thing you see when you open a page with Pyscript while it is loading itself and all the necessary resources.
+
+## Configuration
+
+You can control how `<py-splashscreen>` behaves by setting the value of the  `splashscreen` configuration in your `<py-config>`.
+
+
+| parameter   | default | description |
+|-------------|---------|-------------|
+| `autoclose` | `true`  | Whether to close the splashscreen automatically when the page is ready or not |
+| `enabled` | `true`    | Whether to show the splashscreen or not |
+
+### Examples
+
+#### Disabling the splashscreen
+
+If you don't want the splashscreen to show and log any loading messages, you can disable it by setting `splashscreen.disabled` to `false`.
```

Comment:
I think this is a bug in the TOML parsing (or our handing on it), but this currently throws up a PY1000 error:
```
<py-config>
    splashscreen.disabled = true
</py-config>
```

Although that seems perfectly fine according the the TOML spec and some online TOML parsers... not sure what's going on, but it might be better to omit the `splashscreen.disabled` syntax here until it actually works for us. The TOML example below is solid though.

Reply:
Good idea thanks!